cax/main.py

```diff
@@ -14,12 +16,25 @@ def main():
     parser = argparse.ArgumentParser(description="Copying All kinds of XENON1T data.")
     parser.add_argument('--once', action='store_true',
                         help="Run all tasks just one, then exits")
+    parser.add_argument('--config', action='store', dest='config_file',
+                        help="Load a custom .json config file into cax")    
+
     args = parser.parse_args()
+
     run_once = args.once
 
     # Check passwords and API keysspecified
     mongo_password()
 
+    # Get specified cax.json configuration file for cax:
+    caxjson_config = args.config_file
+    if caxjson_config:
+        if not os.path.isfile( caxjson_config ):
+            logging.error("Config file %s not found", caxjson_config)
+        else:
+            #logging.info("Using custom config file: %s", caxjson_config) # seems to kill rest of output...
```

Comment:
this was trying to warn people of using custom config (as per your previous comment), but for some reason it stopped all the other output info from printing to screen, i'm not sure why